hadoop-hdds/pom.xml

```
@@ -279,41 +279,19 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
         <executions>
           <execution>
             <id>add-classpath-descriptor</id>
-            <phase>package</phase>
+            <phase>prepare-package</phase>
             <goals>
               <goal>build-classpath</goal>
             </goals>
             <configuration>
-              <outputFile>${project.build.directory}/classpath</outputFile>
+              <outputFile>${project.build.outputDirectory}/${pom.artifactId}.classpath</outputFile>
```

Maven outputs a warning for each submodule:

```
The expression ${pom.artifactId} is deprecated. Please use ${project.artifactId} instead.
```

```suggestion
              <outputFile>${project.build.outputDirectory}/${project.artifactId}.classpath</outputFile>
```